mir-module/src/main/resources/xsl/metadata/mir-pdf-errorbox.xsl

```diff
@@ -0,0 +1,171 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xsl:stylesheet xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
+                xmlns:i18n="xalan://org.mycore.services.i18n.MCRTranslation"
+                xmlns:xlink="http://www.w3.org/1999/xlink"
+                xmlns:mcrxsl="xalan://org.mycore.common.xml.MCRXMLFunctions"
+                xmlns:xalan="http://xml.apache.org/xalan"
+                version="1.0" exclude-result-prefixes="i18n mcrxsl">
+
+    <xsl:param name="WebApplicationBaseURL"/>
+    <xsl:param name="ServletsBaseURL" />
+    <xsl:param name="HttpSession"/>
+
+    <xsl:template match="mycoreobject" mode="displayPdfError">
+
+        <xsl:variable name="errorMessages">
+            <xsl:apply-templates select="structure/derobjects/derobject" mode="displayPdfError"/>
+        </xsl:variable>
+
+        <xsl:choose>
+            <xsl:when test="contains($errorMessages,'Clause')">
+                <div class="container pdf-validation mb-3 px-0" id="accordion">
+                    <div class="card-header bg-danger text-white">
+                        <div class="list-group list-group-root well p-3">
+                            <p class="h5">
+                                <xsl:value-of select="i18n:translate('pdf.errorbox.warning.heading')"/>
+                            </p>
+                            <p>
+                                <xsl:value-of select="i18n:translate('pdf.errorbox.warning.message')"/>
+                            </p>
+                        </div>
+                    </div>
+                    <div class="card-body border-left border-right border-bottom">
+                        <xsl:copy-of select="$errorMessages"/>
+                    </div>
+                </div>
+            </xsl:when>
+            <xsl:when test="string-length(normalize-space($errorMessages)) > 0 and not(contains($errorMessages,'Clause'))">
+                <div class="card-header bg-success text-white mb-3">
+                    <div class="list-group list-group-root well p-3">
+                        <p class="h5">
+                            <xsl:value-of select="i18n:translate('pdf.errorbox.success.heading')"/>
+                        </p>
+                        <p>
+                            <xsl:value-of select="i18n:translate('pdf.errorbox.success.message')"/>
+                        </p>
+                    </div>
+                </div>
+            </xsl:when>
+        </xsl:choose>
+    </xsl:template>
+
+    <xsl:template match="structure/derobjects/derobject" mode="displayPdfError">
+        <xsl:variable name="derivateID" select="@xlink:href"/>
+        <xsl:variable name="result" select="document(concat('pdfAValidator:', $derivateID))"/>
```

Comment:
There should be a notNull or exception uri resolver chained in, to prevent the resolver from crashing the whole metadata page like in [MIR-1303](https://mycore.atlassian.net/browse/MIR-1303)

Reply:
I am working on that ticket seperately and will be merging it into this branch in a seperate pull request.